Cargo.toml

```diff
@@ -42,9 +42,9 @@ default-members = [
     "wgpu",
 ]
 
-[workspace.lints.clippy]
-manual_c_str_literals = "allow"
-ref_as_ptr = "warn"
+[workspace.lints.rust]
+manual_c_str_literals = "forbid"
```

Comment:
This is `allow` because c literals are rust 1.77, but our MSRV in wgpu-core/hal/types is 1.76, so we need to suppress it.